spec/support/disability_compensation_form/all_claims_evss_submission.json

```diff
@@ -0,0 +1,157 @@
+{
+    "form526": {
+        "applicationExpirationDate": "2015-08-28T19:53:45+00:00",
+        "claimantCertification": true,
+        "directDeposit": {
+            "accountNumber": "123123123123",
+            "accountType": "CHECKING",
+            "bankName": "SomeBank",
+            "routingNumber": "123123123"
+        },
+        "disabilities": [
+            {
+                "diagnosticCode": 9999,
+                "disabilityActionType": "NEW",
+                "name": "PTSD (post traumatic stress disorder)",
+                "ratedDisabilityId": "1100583",
+                "secondaryDisabilities": [
+                    {
+                        "disabilityActionType": "SECONDARY",
+                        "name": "PTSD personal trauma",
+                        "serviceRelevance": "Lengthy description"
+                    }
+                ]
+            }
+        ],
+        "serviceInformation": {
+            "alternateNames": [
+                {
+                    "firstName": "JKack",
+                    "middleName": "Clint",
+                    "lastName": "Bauer"
+                }
+            ],
+            "confinements": [
```

Comment:
I don't see any `powDisabilities` arrays - are these worth adding? Not sure if they should be within each confinement object or their own property (depends on whether we need to keep the disability list associated with the specific confinement period I guess).

Reply:
Interesting that you brought that up... because thats part of the `specialIssues` work I need to tackle next. Turns out we need to set the `POW` special issue for disabilities that are connected. So i'll need that array to make the match. That change will be part of this ticket https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets.gov-team/13974